src/transform/class/PotentialMethod.js

```diff
@@ -70,4 +78,31 @@ class PotentialMethod {
       replacements: [],
     });
   }
+
+  modifySuperCalls() {
+    const matchSuperConstructorCall = matchesAst({
+      type: 'ExpressionStatement',
+      expression: {
+        type: 'CallExpression',
+        callee: {
+          type: 'MemberExpression',
+          object: obj => isEqualAst(obj, this.potentialClass.superClass),
```

Comment:
Here we're breaking the encapsulation of `PotentialClass` by accessing the `superClass` property directly.